frontend/packages/operator-lifecycle-manager/src/style.scss

```diff
@@ -254,3 +254,55 @@ $co-affinity-row-margin: 15px;
 .olm-descriptor__title {
   display: inline-block;
 }
+
+.pf-c-card__body.co-clusterserviceversion-install__gray_background {
+  background-color: #f0f0f0;
+}
+
+#operator-install-page .pf-c-card {
+  margin-bottom: 5px;
+  max-width: 600px;
+  // margin-top: auto;
+}
+
+#operator-install-page .pf-c-alert {
+  margin-bottom: 5px;
+}
+
+#operator-install-page .pf-c-button.co-clusterserviceversion-link__shift_left {
+  padding-top: 10px;
+  padding-left: 0px;
+}
+
+#operator-install-page .pf-c-button.co-clusterserviceversion-button__left_padding {
+  margin-left: 10px;
+}
+
+.co-operator-install-page__bottom-padding {
```

Comment:
I wouldn't create a class to add custom padding to a `<p>` 
Use PF utility spacing class